src/views/wallet/WalletSettingsScreen.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useMemo, useCallback } from 'react'
+import { Row, Col, Typography, Button, Card, List } from 'antd'
+import { PlusCircleFilled, CloseCircleOutlined } from '@ant-design/icons'
+import { UserAccountType } from '../../types/wallet'
+const { Title, Text, Paragraph } = Typography
+
+// Dummy Data
+const UserAccounts: UserAccountType[] = [
+  {
+    chainName: 'Binancechain',
+    accounts: [
+      {
+        name: 'Main',
+        address: 'tbnb1vxutrxadm0utajduxfr6wd9kqfalv0dg2wnx5y',
+        type: 'internal'
+      },
+      {
+        name: 'Ledger',
+        address: 'tbnb1vxutrxadm0utajduxfr6wd9kqfalv0dg2wnx5y',
+        type: 'external'
+      }
+    ]
+  },
+  {
+    chainName: 'EthereumClassic',
+    accounts: [
+      {
+        name: 'Metamask',
+        address: '0x910286F93b230E221384844C4ae18a14c474E74E',
+        type: 'external'
+      }
+    ]
+  }
+]
+// Dummy data... types not confirmed
+const userAccount = {
+  address: 'tbnb1vxutrxadm0utajduxfr6wd9kqfalv0dg2wnx5y',
+  keystore: {
+    version: 1,
+    id: '84a9e412-90a1-4076-8e7c-e516f78059ee',
+    crypto: {}
+  }
+}
+type ClientTypes = { network: string; chainId: string }
+const UserAccountsScreen: React.FC = (): JSX.Element => {
+  const client: ClientTypes = useMemo(() => {
+    return { chainId: 'testing', network: 'testing' }
+  }, [])
+
+  const downloadLink: string = useMemo(() => {
+    const keystore: string = window.localStorage.getItem('binance') || ''
+    return 'data:text/plain;charset=utf-8,' + encodeURIComponent(keystore)
+  }, [])
+
+  const fileName = (): string => userAccount.address.concat('-keystore.txt')
+
+  const lockWallet = useCallback(async () => {
+    console.log('locking wallet...')
+  }, [])
+  const removeWallet = useCallback(async () => {
+    console.log('removing wallet...')
+  }, [])
+
+  return (
+    <Row>
+      <Title level={3}>Settings</Title>
+      <Col span={24}>
+        <Row gutter={[16, 16]}>
+          <Col md={{ span: 12 }} lg={{ span: 12 }} xl={{ span: 12 }}>
```

Comment:
[nitpick] If `md={{ span: 12 }}` and `md ≥768px`, there is no need to define `lg` + `xl` with same values I guess... See https://ant.design/components/grid/#Col

Reply:
yeah left over from previous alterations. good find.